book/website/reproducible-research/vcs/vcs-resources.md

```diff
@@ -1,5 +1,22 @@
+# Resources
+This chapter covered a lot. Below you'll find:
```

Comment:
```suggestion
This chapter covers a lot. Below you'll find:
```

Reply:
thank you :) 

I may add to this a little more and circle back for a review before pushing to main. Thank you @sgibson91 🌟